README.md

```diff
@@ -74,3 +74,47 @@ fn main() {
     }
 }
 ```
+
+Testing
+-------
+
+The driver test suite is largely composed of a mixture of integration tests and behavioral unit-tests, relying on the official [MongoDB specifications repo](https://github.com/mongodb/specifications). The easiest way to thoroughly test the driver is to set your fork up with TravisCI.
+
+To test the driver locally, you'll need to setup integration tests and specification tests.
+
+> NOTE: Each integration test uses a unique database/collection to allow tests to be parallelized, and will drop their dependencies before running. However, effects are _not_ cleaned up afterwards.
+
+#### Setting up integration tests
+
+All integration tests run on the default MongoDB port, 27017. Before running the tests, ensure that you setup a test database on that port.
+
+Download a version of MongoDB:
```

Comment:
Maybe add a "if you don't already have this installed" parenthetical? I'm guessing the majority of people wanting to this this will be doing it just to verify that that it works on their specific system.